deps/ncrypto/ncrypto.cc

```diff
@@ -4190,6 +4190,36 @@ DataPointer hashDigest(const Buffer<const unsigned char>& buf,
   return data.resize(result_size);
 }
 
+DataPointer xofHashDigest(const Buffer<const unsigned char>& buf,
+                          const EVP_MD* md,
+                          size_t output_length) {
+  if (md == nullptr) return {};
```

Comment:
@codebytere ... can I ask you to take a look at this from a Whether-This-Will-Work-In-BoringSSL perspective?